styx-flyte-client/src/test/java/com/spotify/styx/flyte/client/FlyteAdminClientTest.java

```diff
@@ -104,7 +106,8 @@ public void shouldPropagateCreateExecutionToStub() {
   public void shouldPropagateLabelsAndAnnotationsOnCreateExecutionToStub() {
     var workflowExecution =
         flyteAdminClient.createExecution(PROJECT, DOMAIN, NON_EXISTING_NAME, identifier(NON_EXISTING_NAME),
-            ExecutionMode.SCHEDULED, LABELS, ANNOTATIONS, EXTRA_DEFAULT_INPUTS);
+            ExecutionMode.SCHEDULED, LABELS, ANNOTATIONS, EXTRA_DEFAULT_INPUTS,
+            Collections.emptyMap());
```

Comment:
```suggestion
            Mao.of());
```
shorter this way and maybe it can go in the previous line